inventory/Inventory/settings.py

```diff
@@ -89,22 +89,22 @@
 # Database
 # https://docs.djangoproject.com/en/4.2/ref/settings/#databases
 
-# DATABASES = {
-#     'default': {
-#         'ENGINE': 'django.db.backends.postgresql',
-#         'NAME':  'AirQoHardWare',
-#         'USER': 'postgres',
-#         'PASSWORD': '',
-#         'HOST': 'localhost'
-#     }
-# }
-
-
 DATABASES = {
-    'default' : dj_database_url.parse(os.environ.get('DATABASE_URL'))
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME':  'AirQoHardWare',
+        'USER': 'postgres',
```

Comment:
@OlukaGibson shouldn't these be environmental variables that are different for different ENVIRONMENTS which include:
- Staging
- Development
- Production
cc: @BenjaminSsempala / @Psalmz777